generic_config_updater/generic_updater.py

```diff
@@ -19,7 +19,9 @@ def extract_scope(path):
     if not path:
         raise Exception("Wrong patch with empty path.")
     pointer = jsonpointer.JsonPointer(path)
-    parts = pointer.parts
+
+    # Re-escapes `/` in path parts by converting `/` back to `~1`.
+    parts = [part.replace("/", "~1") for part in pointer.parts]
```

Comment:
>replace [](http://example.com/codeflow?start=18&length=7)

this replace is not enough. Could you use jsonpointer.escape(s) : str->str

Reply:
Discussed offline, we should reuse escape function of jsonpointer to do full escape.